spec/services/vacancy_facets_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
```

Comment:
A new line under the imports helps readability I find (and you did it below too)